client/connhelper/npipe/npipe_other.go

```diff
@@ -0,0 +1,14 @@
+//go:build !windows
+
+package npipe
```

Comment:
nit: I think the idea was that we try as much as possible to abstract the naming so that it's not platform specific (especially for Windows); that's why we'd gone with just a generic package `client`. This `npipe` package is very lean and it's Windows-specific; might be good to go this route only after we've a number of things we are doing with `npipe` other than just one helper.